LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/ResidentialCarePackage.cs

```diff
@@ -16,26 +16,27 @@ public class ResidentialCarePackage
         public Guid Id { get; set; }
 
         /// <summary>
-        /// Gets or sets the Package Id
+        /// Gets or sets the Client Id
         /// </summary>
-        public Guid PackageId { get; set; }
+        public Guid ClientId { get; set; }
 
         /// <summary>
-        /// Gets or sets the Client Id
+        /// Gets or sets the Clients
         /// </summary>
-        public Guid ClientId { get; set; }
+        [ForeignKey(nameof(ClientId))]
+        public Clients Clients { get; set; }
 
         /// <summary>
         /// Gets or sets the Start Date
         /// </summary>
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
These date fields shouldn't be receiving automatic values from the db generated option